catalog/dags/providers/provider_api_scripts/finnish_museums.py

```diff
@@ -95,10 +95,9 @@ def get_media_type(self, record):
 
     def get_batch_data(self, response_json):
         if (
-            response_json is None
+            not response_json
             or str(response_json.get("status")).lower() != "ok"
-            or response_json.get("records") is None
-            or len(response_json.get("records")) == 0
```

Comment:
Why the removal of this last check?

Reply:
The last check is replaced with a `not` check: `not response_json.get('records')` which will return `False` both for `None` and a 0-length list.

Reply:
Oh right, of course! Excellent!